Mathlib/Geometry/Manifold/VectorBundle/SmoothSection.lean

```diff
@@ -227,6 +227,63 @@ lemma ContMDiffOn.smul_section_of_tsupport {s : Π (x : M), V x} {ψ : M → 
     simp [image_eq_zero_of_notMem_tsupport hy, zeroSection]
   · exact Set.compl_subset_iff_union.mp <| Set.compl_subset_compl.mpr ht'
 
+/-- The sum of a locally finite collection of sections is `C^k` iff each section is.
+Version at a point within a set. -/
+lemma ContMDiffWithinAt.sum_section_of_locallyFinite {ι : Type*} {t : ι → (x : M) → V x}
+    (ht : LocallyFinite fun i ↦ {x : M | t i x ≠ 0})
```

Comment:
could you use `support (t i)` instead, here and below?

Reply:
That was the first thing I tried --- it doesn't work since t i is a dependent function.